packages/components/src/components/quickfact/quickfact.stories.ts

```diff
@@ -0,0 +1,249 @@
+import '../../solid-components';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+const { argTypes, parameters } = storybookDefaults('sd-quickfact');
+const { overrideArgs } = storybookHelpers('sd-quickfact');
+const { generateTemplate } = storybookTemplate('sd-quickfact');
+import '../../styles/display/display.css';
+import '../../styles/leadtext/leadtext.css';
+import '../../styles/paragraph/paragraph.css';
+import { html } from 'lit';
+import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
+import { userEvent } from '@storybook/test';
+import { waitUntil } from '@open-wc/testing-helpers';
+import { withActions } from '@storybook/addon-actions/decorator';
+
+export default {
+  title: 'Components/sd-quickfact',
+  component: 'sd-quickfact',
+  args: overrideArgs([
+    {
+      type: 'slot',
+      name: 'default',
+      value: `<div class="slot slot--border slot--text h-12">Default slot</div>`
+    },
+    {
+      type: 'slot',
+      name: 'summary',
+      value: `<div slot="summary"> <p class="text-base font-normal leading-normal  sm:text-3xl sm:leading-tight">Lorem Ipsum</p>
+            <div class="text-base font-normal leading-normal sm:text-xl">Con sectetur adipiscing elit</div></div>`
+    }
+  ]),
+
+  argTypes,
+  parameters: { ...parameters },
+  decorators: [withActions] as any
+};
+/**
+ * This shows sd-quickfact in its default state.
+ */
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({ args });
+  }
+};
+
+/**
+ * This shows sd-quickfact is displayed differently depending on whether or not the default slot is filled.
+ */
+
+export const OpenXContent = {
+  name: 'Open x Content',
+  render: (args: any) => {
+    return generateTemplate({
+      axis: {
+        y: {
+          type: 'slot',
+          name: 'default',
+          values: [
+            {
+              title: 'with content',
+              value: `<div class="slot slot--border slot--text h-12">Default slot</div>`
+            },
+            {
+              title: 'without content',
+              value: ''
+            }
+          ]
+        },
+        x: {
+          type: 'attribute',
+          name: 'open',
+          values: [false, true]
```

Comment:
even though I see that this might not be part of this story, but it still looks "buggy/wrong" if they are not horizontally aligned:
<img width="867" alt="Screenshot 2024-07-02 at 19 31 27" src="https://github.com/solid-design-system/solid/assets/48211526/1f246b33-f1d3-4416-8f43-161ab5a18072">
Especially as they are jumping when changing the open state.